makefile

```diff
@@ -0,0 +1,7 @@
+all: black
+
+black:
+	black .
+
+requirements:
```

Comment:
You can run this by typing `make requirements`. It will run the `pip` command on line 7 for you.

The `makefile` is also a great way to keep track of complicated command-line incantations.

Reply:
Hmmm, I merged this pull request and updated locally in pycharm, but it apparently ate my decal images (pictures of plants and trees and such my son made, for fluffing up the background). So... I had to revert. I will see if I can try to merge the changes while keeping the decal images and associated random generated passive sprites.... this is all a bit new for me so I am probably doing dumb stuff ;)

Reply:
That's weird. Shouldn't happen. I'll have a look as well.